lttng/org.eclipse.tracecompass.lttng2.kernel.ui.swtbot.tests/src/org/eclipse/tracecompass/lttng2/kernel/ui/swtbot/tests/HelpMessageTest.java

```diff
@@ -78,7 +78,15 @@ public void testHelpMenus() {
             shell.activate();
             SWTBot helpShellBot = shell.bot();
             assertEquals("Help", helpShellBot.canvas().getText());
-            assertEquals(entry.getValue(), helpShellBot.label(1).getText());
+
+            int i = 0;
+            String text = helpShellBot.label(i).getText();
+            // Find the label out of 3 labels that contains the expected message
+            while (!text.equals(entry.getValue()) && i <= 2) {
+                i++;
```

Comment:
won't this skip over value zero? Move one line down?

Reply:
nvm - you get it up there

Reply:
no, the first time it will check index 0. If it is equal then it goes over the while loop. Anyways, I changed the implementation in the next update of this PR.